recipes/qt/6.x.x/conanfile.py

```diff
@@ -16,7 +16,19 @@ def filename(self):
 
     @property
     def content(self):
-        return "[Paths]\nPrefix = %s\n" % self.conanfile.deps_cpp_info["qt"].rootpath.replace("\\", "/")
+        return """[Paths]
```

Comment:
Not sure which is right, but this is not coherent with https://github.com/conan-io/conan-center-index/blob/master/recipes/qt/6.x.x/conanfile.py#L506
We should probably declare a dedicated function generating this content, and call it from `def content(self):` and `def package(self):`

Reply:
Good catch! Missed that difference from 5.x to 6.x.

Reply:
I think the 5.x should change the dir to `res` too, since it makes common `imports` to import plugins into the bin folder which is redundant.

Reply:
By using [`textwrap.dedent`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent), you can keep indentation of the python source sane.
e.g.
```python
textwrap.dedent("""\
    line1
    line2
    line3""")
```